daemon/logger/factory.go

```diff
@@ -46,7 +53,7 @@ func (lf *logdriverFactory) get(name string) (Creator, error) {
 
 	c, ok := lf.registry[name]
 	if !ok {
-		return c, fmt.Errorf("logger: no log driver named '%s' is registered", name)
+		return c, fmt.Errorf("unsupported log driver '%s'", name)
```

Comment:
perhaps "log driver .. not found"? Because it may be *supported*, just not installed. Just thinking out loud here